src/libs/error.js

```diff
@@ -1,18 +1,15 @@
 import _ from 'lodash/fp'
-import { reloadAuthToken, signOut } from 'src/libs/auth'
-import { notify, sessionTimeoutProps } from 'src/libs/notifications'
+import { notify } from 'src/libs/notifications'
 
 
 export const reportError = async (title, obj) => {
-  if (obj instanceof Response && obj.status === 401) {
-    if (!await reloadAuthToken()) {
-      notify('info', 'Session timed out', sessionTimeoutProps)
-      signOut()
-    }
-    // Don't report an error if we've successfully reloaded the auth token
-  } else {
-    notify('error', title, { detail: await (obj instanceof Response ? obj.text() : obj) })
+  // Do not show an error notification when a session times out.
+  // Notification for this case is handled elsewhere.
```

Comment:
We show a "Session timed out" info notification instead of error notifications in this case.